Pokedex_iosTests/AdvancedFilterTests.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  AdvanceFilterTests.swift
+//  Pokedex_iosTests
+//
+//  Created by Alvaro Choque on 11/7/22.
+//
+
+import Foundation
+
+import XCTest
+@testable import Pokedex_ios
+
+class AdvancedFilterTests: XCTestCase {
+    let mockTest = MockPokedexService()
+    let viewModel = PokemonListViewModel(pokedexService: MockPokedexService())
+    
+    func testGetPokemonsFilteredByTest() {
+//        viewModel.getPokemons()
```

Comment:
remove not needed commented code (or again if its not yours discard changes that are not yours)